File: app/src/main/java/com/example/weizhenbin/wangebug/tools/UUIDTool.java
package com.example.weizhenbin.wangebug.tools;

import java.util.UUID;

/**
 * Created by weizhenbin on 2018/9/21.
 */

public class UUIDTool {
    public static String getUUID(){
        return UUID.randomUUID().toString().replaceAll("-", "");
    }
}

File: app/src/main/java/com/example/weizhenbin/wangebug/tools/UUIDTool.kt
package com.example.weizhenbin.wangebug.tools

import java.util.*

/**
 * Created by weizhenbin on 2018/9/21.
 */

object UUIDTool {
    val uuid: String
        get() = UUID.randomUUID().toString().replace("-".toRegex(), "")
}
